pkg/component/operator/image/v0/task_draw_classification_test.go

```diff
@@ -0,0 +1,62 @@
+package image
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+
+	_ "embed"
+
+	"google.golang.org/protobuf/types/known/structpb"
+
+	"github.com/instill-ai/pipeline-backend/pkg/component/base"
+)
+
+//go:embed testdata/cls-dog.json
+var clsDogJSON []byte
+
+// TestDrawClassification tests the drawClassification function
+func TestDrawClassification(t *testing.T) {
+	testCases := []struct {
+		name      string
+		inputJSON []byte
+	}{
+		{
+			name:      "Classification Dog",
+			inputJSON: clsDogJSON,
+		},
+	}
+
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+			inputData := &structpb.Struct{}
+			if err := json.Unmarshal(tc.inputJSON, inputData); err != nil {
+				t.Fatalf("Failed to unmarshal test data: %v", err)
+			}
+
+			bc := base.Component{}
+			component := Init(bc)
+
+			e, err := component.CreateExecution(base.ComponentExecution{
+				Component: component,
+				Task:      "TASK_DRAW_CLASSIFICATION",
+			})
+
+			if err != nil {
+				t.Fatalf("drawClassification create execution returned an error: %v", err)
+			}
+
+			ir, ow, eh, job := base.GenerateMockJob(t)
```

Comment:
💭 I don't love this function being in the `base` package, and part of the production code. The ideal place would be `internal/mock/helper.go` or `base/test/mockhelper.go`. The former isn't possible right now due to an import cycle but I think we can break the cycle easily using `base_test` as the testing package in the `/base` directory.

Reply:
addressed [here](https://github.com/instill-ai/pipeline-backend/pull/685/commits/e8f77d00003a3aa9a6e8d3b55f596d691ba5b6e8)

Reply:
Agree with this. Since the `component` codebase is now living together with the `pipeline-backend` codebase, there are some package structure will need to be adjusted accordingly. cc'd @donch1989 @chuang8511.